### PR TITLE
Improved command usage

### PR DESCRIPTION
**Pull request by LcyDev**
Currently, the command usage is different from the familiar one.

Admin commands get opaqued by the user ones and the list of players `(lite.eco.suggestion.players)`

This makes it more complicated to use.

My current suggestions:
 * Change all the admin commands to a dedicated command `/eco`
 * Maybe change the permissions to `lite.eco.admin.{sub}`? but this could break permissions
 (Nothing too chaotic, users will just need to reconfigure)

User improvements:
 * Add a proxied command `/pay`, faster to use.
 (I'm having an issue with this https://github.com/Incendo/cloud/issues/426)

### CODE REVIEW

File: src/main/kotlin/encryptsl/cekuj/net/commands/MoneyCMD.kt [hunk 1]
```diff
@@ -131,13 +131,19 @@ class MoneyCMD(private val liteEco: LiteEco) {
...
-        player: Player,
+        commandSender: CommandSender,
         @Argument(value = "player", suggestions = "offlinePlayers") offlinePlayer: OfflinePlayer,
         @Argument(value = "amount") @Range(min = "1.00", max = "") amount: Double
     ) {
+        if (commandSender !is Player) { // temp fix
+            commandSender.sendMessage(ModernText.miniModernText("<red>TerminalConsoleCommandSender is not allowed to execute that command. Must be of type Player"))
+            return
+        }
+        val player: Player = commandSender
```
Comment (EncryptSL) [inline src/main/kotlin/encryptsl/cekuj/net/commands/MoneyCMD.kt:146]: Seems good for me but idk if this really necessary. 
But for devs who don't know how Cloud framework works, can be better this.

**Review by EncryptSL — APPROVED** [review]

**Review by EncryptSL — APPROVED** [review]
Okay, can be

**Comment by LcyDev** [comment]
I'm starting to learn Cloud frameworks, i'm pushing a fix for that

**Comment by EncryptSL** [comment]
Its okay :). 👍

**Comment by LcyDev** [comment]
I finally have internet again, but i'm not sure what to do.
I can't find stuff about the framework other than the github itself.

I think making an CommandExceptionHandler and registering the exceptions there could work.
But i'm not sure if doing that is necessary or if this is a bug with cloud

For now i just inverted the condition and changed the message.

**Pull request merged**